ls1intum/Artemis
# Bugfix/deactivate compass

### Motivation and Context
Deactivating compass until hazelcast problem is resolved

### Description
Compass is unusable right now since hazelcast is doing too much data transfer which makes Artemis unusable

The PR also adds audit log entries when users with the editor role change an exercise.

### Steps for Testing
1. Log in to Artemis
2. Create a modeling exercise (Class Diagram or Activity Diagram)
3. It shouldn't be possible to activate compass. Following checkbox shouldn't appear
   - ![image](https://user-images.githubusercontent.com/16825065/116912823-01ef9280-ac49-11eb-8975-f06ea9b121d9.png)
4. Login as students
5. Submit similar solutions to modeling exercise with different students
6. Assess the submissions 
7. No suggesstions must be made to similar element
